### ISSUE TITLE
parse-md update needed: unable to render aria-disabled, etc. attributes if using button icon in markdown

### ISSUE THREAD

**Issue by silverli** [tmobile/magentaA11y#341]
We learned this after trying to work on https://github.com/tmobile/magentaA11y/issues/232#issuecomment-3392312112 (see history there)

**Comment by joe-watkins** [comment]
#340 solved this ;)
